City-of-Helsinki/helsinki-paatokset
# PP-199: Decision and case content

**To test**
- Checkout branch
- Run `make new`
- SSH into the container and run `drush ap:fs;drush mim ahjo_decisions:latest;drush mim ahjo_cases:latest;drush mim ahjo_meetings:latest;drush mim ahjo_decisionmakers:all;drush ap:ud --localdata -v`
- Test a few decisions from this list: https://helsinki-paatokset.docker.so/fi/admin/content/decisions
  - Multiple decisions and meeting data: https://helsinki-paatokset.docker.so/fi/asia/hel-2020-006250/ecf8dfac-947c-4dae-a603-132e8c207ded
  - Also test a few decisions without diary numbers. The layout might have some issues that will be solved in a separate ticket, but there shouldn't be any fatal errors, etc
- The content and motion HTML should be parsed into the main content section and accordion elements
- The dropdown navigaation should work through AJAX, and should update the "download decision PDF" and "other decisions for the meeting" links
- Check that the agenda list links work: https://helsinki-paatokset.docker.so/fi/paattajat/tarkastuslautakunta/asiakirjat/U003110202118
  - All links with an arrow should take you to a decision
- Read the code changes. Is there anything that doesn't make sense or should be refactored?
